reference/6/About/about_Escape_Characters.md

```diff
@@ -119,22 +119,49 @@ Escape Sequence | Special Character
 `0 | Null
 `a | Alert
 `b | Backspace
+`e | Escape
 `f | Form feed
 `n | New line
 `r | Carriage return
 `t | Horizontal tab
+`u{x} | Unicode escape sequence for characters with hex value x (up to xxxxxx)
 `v | Vertical tab
 
-For example:
+This next example uses new line and tab characters to format the string:
 
 ```powershell
-"12345678123456781`nCol1`tColumn2`tCol3"
+"12345678123456781`nCol1`tColumn2`tCol3`u{2195}"
 ```
 
 ```output
-# 12345678123456781
+12345678123456781
+
+Col1 Column2 Col3↕
+```
+
+This next example uses the escape character to specify a virtual
+terminal sequence to output text with the foreground color of Orange.
+This example requires Windows 10 Anniversary Update or a Linux or
+macOS terminal that supports 24-bit color.
 
-Col1 Column2 Col3
+```powershell
```

Comment:
Perhaps choose a simpler example and one that also works in the stock macOS terminal? Something along the lines of:

```powershell
$fgColor = 32 # green - see http://en.wikipedia.org/wiki/ANSI_escape_code
"It ain't easy being `e[${fgColor}mgreen`e[0m."
```